docs/topics/loaders.rst

```diff
@@ -173,17 +177,17 @@ The other thing you need to keep in mind is that the values returned by input
 processors are collected internally (in lists) and then passed to output
 processors to populate the fields.
 
-Last, but not least, Scrapy comes with some :ref:`commonly used processors
-<topics-loaders-available-processors>` built-in for convenience.
+Last, but not least, ``itemloaders`` comes with some :ref:`commonly used
+processors <processors>` built-in for convenience.
```

Comment:
```suggestion
Last, but not least, itemloaders_ comes with some :ref:`commonly used
processors <itemloaders:built-in-processors>` built-in for convenience.
```

Note: ``` :doc:`itemloaders <itemloaders:index>` ``` could work as an alternative to `itemloaders_` that links to the home page of the `itemloaders` documentation.